src/main/java/AddCommand.java

```diff
@@ -0,0 +1,29 @@
+package duke;
```

Comment:
Any reason you packaged all under a singular duke package instead of splitting the different classes into more packages? i.e. duke.commands, duke.entities